<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostWidget</class>
 <widget class="QWidget" name="PostWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="commentButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>45</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>评论</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 1px solid gray; 
    border-radius: 10px;    
    padding: 5px;           
    background-color: white; /* 明确设置背景为白色 */
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>45</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>收藏</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
